### PR TITLE
Drop support for Python 3.4, add support for Python 3.8.

### PR DESCRIPTION
**Pull request by icemac**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,10 +2,10 @@ language: python
 dist: xenial
 python:
   - "2.7"
-  - "3.4"
   - "3.5"
   - "3.6"
   - "3.7"
+  - "3.8-dev"
```
Comment (mgedmin) [inline .travis.yml]: You can use 3.8 final now.

Comment (jugmac00) [reply .travis.yml]: This seems undocumented? Have not found Travis documentation updated.

Comment (mgedmin) [reply .travis.yml]: https://travis-ci.community/t/add-python-3-8-support/5463/5 is where Python 3.8 was requested and notified (in a very cryptic way that links to some internal Travis job that builds and uploads Python versions into the S3 bucket that Travis jobs then use to download them).

https://docs.travis-ci.com/user/reference/xenial/#python-support says "3.4 or higher", and, well, 3.8 is higher ;)  But it is annoying that you cannot easily see an up-to-date list of actually supported Python versions.

Comment (mgedmin) [reply .travis.yml]: Oh, and I have personally tried 3.8 in my projects this morning and verified that it works:
https://github.com/mgedmin/check-python-versions/commit/af981a15b7a4682bc8381c43bcdf8553fc08eff1

Comment (icemac) [reply .travis.yml]: Thank you for testing.

**Pull request merged**